spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -948,10 +948,8 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
 
   private def castTest(input: DataFrame, toType: DataType): Unit = {
 
-    // we do not support the TryCast expression in Spark 3.2 and 3.3
+    // we now support the TryCast expression in Spark 3.2 and 3.3
     // https://github.com/apache/datafusion-comet/issues/374
```

Comment:
nit: I don't think there is value in linking to an issue that is now resolved. I would just remove this comment

Reply:
removed.